src/Standards/PSR12/Docs/Classes/ClosingBraceStandard.xml

```diff
@@ -0,0 +1,35 @@
+<documentation title="Closing Brace">
+    <standard>
+    <![CDATA[
+    Any closing brace must not be followed by any comment or statement on the same line.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: No content on the line with the closing brace.">
```

Comment:
```suggestion
        <code title="Valid: Closing brace is the last content on the line.">
```

This particular PSR 12 rule does not imply anything about content _before_ the closing brace, so purely based on the rule and how the sniff is checking this rule, the below code would be perfectly valid, while with the description as it was, it would not be allowed (even though the sniff doesn't check content before the close brace).

```php
function foo() { echo 'Hello'; }
```